CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+
+- Marsha: cronjob running check_harvested management command
```

Comment:
Don't forget to port this PR to @madmatah's branch https://github.com/openfun/arnold/tree/migration-to-k8s to avoid a regression.

Reply:
Yes I plan to do it.